internal/ingress/controller/config/config.go

```diff
@@ -137,9 +135,9 @@ type Configuration struct {
 	// By default access logs go to /var/log/nginx/access.log
 	AccessLogPath string `json:"access-log-path,omitempty"`
 
-	// HttpAccessLogPath sets the path of the access logs for http context globally if enabled
+	// HTTPAccessLogPath sets the path of the access logs for http context globally if enabled
 	// http://nginx.org/en/docs/http/ngx_http_log_module.html#access_log
-	HttpAccessLogPath string `json:"http-access-log-path,omitempty"`
+	HTTPAccessLogPath string `json:"http-access-log-path,omitempty"`
```

Comment:
As this is an exported field and part of an API, I'm wondering if we want to change it right now or keep it as is and ignore the error.

@tao12345666333 deferring to you

Reply:
Disregard,this is internal package so far so we are good.

BTW, please make sure this field is not being used also on templates, etc